i3lock.h

```diff
@@ -2,6 +2,7 @@
 #define _I3LOCK_H
 
 // boy i sure hope this doesnt change in the future
+#include <cairo/cairo.h>
 #define NANOSECONDS_IN_SECOND 1000000000
```

Comment:
Nitpick: the include got in between the comment and the thing it is commenting

Reply:
Oh, sure, I'll move it above.